pkgs/native_assets_cli/lib/src/json_utils.dart

```diff
@@ -22,17 +22,13 @@ extension JsonUtils on Map<String, Object?> {
   core.int int(String key) => get<core.int>(key);
   core.int? optionalInt(String key) => getOptional<core.int>(key);
 
-  Uri path(String key,
-          {required Uri baseUri,
-          bool resolveUri = true,
-          bool mustExist = false}) =>
-      _pathToUri(get<String>(key), baseUri: baseUri, resolveUri: resolveUri);
+  Uri path(String key, {bool resolveUri = true, bool mustExist = false}) =>
```

Comment:
Shouldn't the `resolveUri` and `baseUri` param be removed?

Reply:
Nicely spotted, yes! Done.